cli/flags.rs

```diff
@@ -1009,6 +1010,11 @@ fn test_subcommand<'a, 'b>() -> App<'a, 'b> {
         .help("Don't return error code if no test files are found")
         .takes_value(false),
     )
+    .arg(
+      Arg::with_name("terse")
+        .long("terse")
+        .help("UNSTABLE: Display one character per test instead of one line"),
```

Comment:
Can you add here that `--terse` implies `--quiet`?

Reply:
Ended up not doing that after all. 
Aiming to make quiet only applies to diagnostics and terse only applies to the reporter(s).